packages/dappmanager/src/modules/chains/drivers/bitcoin.ts

```diff
@@ -35,7 +35,14 @@ export async function bitcoin(
 ): Promise<ChainDataResult> {
   const container = dnp.containers[0];
   if (!container) throw Error("no container");
-  const containerDomain = getPrivateNetworkAlias(container);
+
+  const {dnpName,serviceName} = container;
```

Comment:
In order to avoid having to duplicate this code in every driver, I would add a function called `build/generateNetworkAliasByPkg({service: string, pkg: InstalledPackageData})`

It would check if the package is monoservice or the specified service is main to set the value of `isMainOrMonoservice`

Then, it would call `build/generateNetworkAlias` to get an available alias